test/gtest/uct/ib/test_ib.cc

```diff
@@ -445,9 +445,19 @@ class test_uct_ib_sl : public test_uct_ib_with_specific_port {
 };
 
 UCS_TEST_P(test_uct_ib_sl, check_ib_sl_config) {
+    const char *max_avail_sl_str = getenv("GTEST_MAX_AVAIL_SL");
+    uint8_t sl, max_avail_sl;
+
+    if (max_avail_sl_str == NULL) {
+        max_avail_sl = UCT_IB_SL_NUM;
+    } else {
+        max_avail_sl = (uint8_t)strtoul(max_avail_sl_str, NULL, 10);
```

Comment:
IMO the last param to strtoul can be "0" so it would detect the base automatically